### ISSUE TITLE
Antennapod support

### ISSUE THREAD

**Issue by neelusR** [aghontpi/ad-silence#126]
Hi, it would be great of you could add AntennaPod support.

**Comment by voidplayer** [comment]
Yes, please, those ads inserted on the downloaded podcasts are very annoying and hard to fight since they are randomly injected on the podcast. They just start playing mid sentence blasting the volume. Very annoying

**Comment by aghontpi** [comment]
@neelusR @voidplayer 

sry for the late reply couldn't get to work on this for sometime, the latest version of the app has custom app support, you can add/ configure and also debug it with the new debug view,

 get the latest release from https://github.com/aghontpi/ad-silence/releases or playstore, fdroid might take some time to publish.

for more you can check here: https://github.com/aghontpi/ad-silence/pull/282 or 
https://github.com/aghontpi/ad-silence?tab=readme-ov-file#custom-apps

**Comment by neelusR** [comment]
Hi @aghontpi, thanks for the update I will download it and try it.

**Comment by aghontpi** [comment]
closing this, as of version 0.7.x and up,  users can now add custom apps themselves.

Add custom app:
https://github.com/aghontpi/ad-silence?tab=readme-ov-file#custom-apps
https://github.com/aghontpi/ad-silence/wiki/Custom_App_Wiki

debug log:
https://github.com/aghontpi/ad-silence?tab=readme-ov-file#debug-log
